src/main/java/org/reaktivity/nukleus/tcp/internal/TcpConfiguration.java

```diff
@@ -22,6 +22,7 @@
     public static final String MAXIMUM_BACKLOG_PROPERTY_NAME = "nukleus.tcp.maximum.backlog";
     public static final String WINDOW_THRESHOLD_PROPERTY_NAME = "nukleus.tcp.window.threshold";
     public static final String MAX_CONNECTIONS_NAME = "nukleus.tcp.max.connections";
+    public static final String TCP_KEEP_ALIVE_NAME = "nukleus.tcp.keepalive";
```

Comment:
Rename constant to `KEEPALIVE_NAME` for consistency with naming pattern for other constants.